doc/mgr/prometheus.rst

```diff
@@ -199,8 +253,8 @@ prometheus.yml
 ::
 
     global:
-      scrape_interval:     15s
-      evaluation_interval: 15s
+      scrape_interval:     10s
+      evaluation_interval: 10s
```

Comment:
This should be changed back to 15, we already use the scraping interval of 15 seconds when deploying prometheus server with ceph-ansible and we should keep this in sync.

Reply:
Does ceph-ansible use 15 seconds for scraping data from node-exporters, too?

Reply:
Sorry, I'm asking because it is recommended to use the same scrape interval for different sources to avoid alignment issues. I also wonder why 15 seconds are used as 10 seconds would increase the resolution of the data. Is there any limitation that prevents us from using 10 seconds here? 

Is ceph-ansible used in production or just for testing?

Reply:
Yes, we use the same scrape interval for all the exporters there:

https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-prometheus/templates/prometheus.yml.j2

IIRC, we have been hitting issues with customers needing more than 10 seconds for scraping quite frequently. We have already had some issues with customers having to increase the scraping interval because of the number of the nodes in their clusters but setting the scrape interval to 15 seconds looked like a sane default value in most cases.

Reply:
> We have already had some issues with customers having to increase the scraping interval because of the number of the nodes in their clusters but setting the scrape interval to 15 seconds looked like a sane default value in most cases.

I'll change it back to 15s, though may you tell me if those issues arose due to other exporters or this one? Because this implementation should at least mitigate such issues in this exporter.

Reply:
IIRC, we were hitting issues with this module/exporter, especially when there were other ceph-mgr modules running, machines were underpowered or there was just too much data/nodes in the cluster.

I suppose this implementation should help with the first case since it creates a separate thread for scraping metrics.